spring-cloud-task-docs/src/main/asciidoc/features.adoc

```diff
@@ -212,3 +212,39 @@ marking the final state of the task.
 }
 ```
 
+=== Launching a task from a Spring cloud stream
+
+Allows a user to launch tasks from a stream.  This is done by creating a sink that
+listens for a message that contains a `TaskLaunchRequest` as a part of its payload.
```

Comment:
"as a part of its payload" or "as the payload"?  If you got something other than a `TaskLaunchRequest`...would it work?